source/js/components/pulse-project-list/pulse-project-list.jsx

```diff
@@ -63,10 +63,24 @@ class PulseProjectList extends Component {
     projectXHR.send();
   }
 
+  // Giving users ability to link to pulse objects using an anchor link
+  // and having it render in the right place after recieving data.
+  scrollToLinkedPulseObject(){
+    const idOfLinkedPulseObject = (window.location.hash).substr(1)
+    const linkedPulseObject = document.getElementById(idOfLinkedPulseObject);
```

Comment:
Note that fragment ids are literally their own query selector, so we can just do this:
```suggestion
    const linkedPulseObject = document.querySelector(window.location.hash);
```